tests/testthat/tests-get_dataset.R

```diff
@@ -10,8 +10,15 @@ test_that("download tab from DOI and filename", {
   actual    <- get_dataset(contents[[ds_index]])
   files     <- actual$files
   expected_dv <- retrieve_info_dataverse("expected-dataverse.yml")
+  expected_names <- c(
+    "citationDate", "createTime", "datasetId",
+    "datasetPersistentId", "fileAccessRequest", "files", "id",
+    "lastUpdateTime", "latestVersionPublishingState", "license",
+    "metadataBlocks", "publicationDate", "releaseTime",
+    "storageIdentifier", "UNF", "versionMinorNumber",
+    "versionNumber", "versionState")
 
-  expect_length(actual                      , 17L)
```

Comment:
This test started to fail, and blocked R CMD check from successful completion. I am unsure of the reason for this length change -- a simple addition of an existing field, or more complicated. I assume that it is not important for the existing code, but provide a more explicit test so that changes are more obvious moving forward.